### DISCUSSION TITLE
[2023-04-24] fsic-10th-meeting

### DISCUSSION THREAD

**Discussion by kevinjantw** [bol-edu/caravel-soc#35] [General]
[fsic-10th-meeting 4-24-2023.pdf](https://github.com/bol-edu/caravel-soc/files/11308852/fsic-10th-meeting.4-24-2023.pdf)

**Comment by TonyHo722** [comment]
add link for video 全端 IC 設計工程師伴學松 第十次 https://www.youtube.com/watch?v=1gmHZmOV_Mg&t=1511s
